### PR TITLE
Olu oyedipe bio

### PR DESCRIPTION
**Pull request by OluOyedipe**
Adding the bio page. Please review my changes.

### CODE REVIEW

File: bios/OluOyedipe.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Olu Oyedipe
+
+**Name:** Olu Oyedipe
+
+**Occupation:** Snake Wrangler
+
+**Location:** Amazon Jungle
```
Comment (OluOyedipe) [inline bios/OluOyedipe.md:7]: Is this the jungle in brazil or the jungle in seatle

File: bios/OluOyedipe.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Olu Oyedipe
+
+**Name:** Olu Oyedipe
```
Comment (OluOyedipe) [inline bios/OluOyedipe.md:3]: Who is this

Comment (huynhmark) [inline bios/OluOyedipe.md:3]: What's a Snake Wrangler?

Comment (OluOyedipe) [inline bios/OluOyedipe.md:3]: Someone who finds new exotic snakes and sells them to Biotech firms

**Pull request merged**